packages/cli/src/commands/devices.ts

```diff
@@ -84,7 +108,16 @@ export default class DevicesCommand extends APICommand<typeof DevicesCommand.fla
 				}
 				return devices
 			},
-			id => this.client.devices.get(id),
+			async (id) => {
+				const device = await this.client.devices.get(id, deviceGetOptions)
+				// Note -- we have to do this explicitly because the API does not honor the includeHealth parameter
+				// for individual devices
+				if (this.flags.health) {
```

Comment:
This could be refactored to be a parallel call since we aren't using the device to get the health state.

Reply:
Done